packages/examples/src/wrapperTs.ts

```diff
@@ -1,18 +1,18 @@
-import { disposeEditor, startEditor, swapEditors, updateModel, wrapper } from './common.js';
-
+import getKeybindingsServiceOverride from '@codingame/monaco-vscode-keybindings-service-override';
 import 'monaco-editor/esm/vs/basic-languages/typescript/typescript.contribution.js';
 import 'monaco-editor/esm/vs/language/typescript/monaco.contribution.js';
-
-import { buildWorkerDefinition } from 'monaco-editor-workers';
+import { disposeEditor, startEditor, swapEditors, updateModel, wrapper } from './common.js';
 import { UserConfig } from 'monaco-editor-wrapper';
+import { buildWorkerDefinition } from 'monaco-editor-workers';
+
 buildWorkerDefinition('../../../node_modules/monaco-editor-workers/dist/workers', import.meta.url, false);
 
-const codeUri = '/tmp/hello.ts';
+const codeUri = '/workspace/hello.ts';
```

Comment:
Was there a reason for changing from tmp to workspace for all these cases?

Reply:
`/tmp` felt temporary. 😆  `/workspace` is somewhat nicer, IMO.